app/index.html

```diff
@@ -39,25 +39,24 @@
               <div class="col-md-5 titleDiv">
                 <!-- Logo -->
                 <div class="logo">
-                  <h1><a href="#/transactions"><img src="images/openhim-logo-green.png" alt="Logo" style="width: 140px; margin-right: 20px;" />{{appTitle}}</a></h1>
-                  <!--<h1><a href="#/transactions"><i class="glyphicon glyphicon-cloud"></i>{{appTitle}}</a></h1>-->
+                  <h1><a href="#/transactions"><img src="images/openhim-logo-green.png" alt="Logo" />{{appTitle}}</a></h1>
                 </div>
               </div>
 
               <div ng-show="navMenuVisible === true">
                 <div class="" style="float: right">
-                  <div class="navbar navbar-inverse" role="banner">
-                      <nav class="navbar-collapse bs-navbar-collapse navbar-right" role="navigation">
-                        <ul class="nav navbar-nav">
-                          <li class="dropdown">
-                            <a href="" class="dropdown-toggle" data-toggle="dropdown"><i class="glyphicon glyphicon-user"></i><span ng-bind="sessionUser"></span><b class="caret"></b></a>
-                            <ul class="dropdown-menu animated fadeInUp">
-                              <li><a href="#/profile">Profile</a></li>
-                              <li><a href="#/logout">Logout</a></li>
-                            </ul>
-                          </li>
-                        </ul>
-                      </nav>
+                  <div class="navbar navbar-inverse" role=" banner" style="float: right;">
```

Comment:
Correct, I added inline style at first as its a bit easier to get the layouts right. Must still move most of the styles into CSS classes